feat(changelog): iam, em and s3 (#1076)
scaleway/docs-content: changelog/iam/december2022/2022-12-07-iam-general-availibility.mdx

---
title: Identity and Access Management (IAM) is now in General Availability!
status: added
author:
    fullname: 'Join the #iam channel on Slack.'
    url: 'https://slack.scaleway.com/'
date: 2022-12-07
category: iam
product: iam
---

IAM allows you to share access to the management of your Scaleway resources, in a controlled and secure manner. 

> You can now manage policies to give access to Scaleway users, applications and groups. API keys can be attached to a user or an application and have the same rights.

Reviewer: ```suggestion
You can now create and manage policies to give fine-grained access rights to Scaleway users, applications and groups. API keys can be attached to a user or an application and inherit their rights.
```

> IAM Features are available though Console, CLI and Terraform.

Reviewer: ```suggestion
IAM features are available though Console, CLI and Terraform.
```

> For organizations created before December 5th, IAM can be activated by the Owner of the organization on the console. Accounts created since December 5th 2022 have IAM activated by default.

Reviewer: ```suggestion
For Organizations created before December 5th, IAM can be activated by the Owner of the Organization on the console. Accounts created since December 5th 2022 have IAM activated by default.
```
